### PR TITLE
thenextweb.com webpage clone

### PR DESCRIPTION
**Pull request by ebeagusamuel**
In this project, we built a responsive clone of the web page of thenextweb.com.
The project is meant to be an avenue for us to practice building responsive web designs.
It was built with only html and css.

### CODE REVIEW

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+![screenshot](./images/Screenshot1.png)
+![screenshot](./images/Screenshot2.png)
+![screenshot](./images/Screenshot3.png)
+![screenshot](./images/Screenshot4.png)
+![screenshot](./images/Screenshot5.png)
+![screenshot](./images/Screenshot6.png)
+![screenshot](./images/Screenshot7.png)
+![screenshot](./images/Screenshot8.png)
+![screenshot](./images/Screenshot9.png)
+
```
Comment (christianotieno) [inline README.md]: These images are not showing. Kindly fix this, thank you.

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** 
Hey @ebeagusamuel & @Abdoulaye-Thespy  
-
You have done a splendid job!  Keep it up.
We, however, do have some minor changes to carry out before we proceed (see review comment).
##
- Kindly, add the date and change the gradient colour of the main article to match the theme of the tnw page ([link to page](https://thenextweb.com/))
- Current: 👇 
tnw-clone
- Expected: 👇 
tnw
- FIx the footer social links to also match the original page, social links theme.
- Current: 👇 
social-clone
- Expected: 👇 
social

Please make these changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. 
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding 
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by thsvr** [comment]
### **STATUS: PROJECT APPROVED** :100: :1st_place_medal: 

Hi @ebeagusamuel and @Abdoulaye-Thespy :smile_cat: 

### **Congrats on this great job!** 
You guys did the requested changes asked by Christiano(TSE) and your project is approved :fireworks: :fireworks: :fireworks: :fireworks:


- You guys got the main idea of the name of the features branches :trophy: :1st_place_medal: 

If you have any questions, don't hesitate to contact me through slack [@thaís Vieira Azevedo](url)

### **Happy coding and good luck on the next project** :computer: 

_Project reviewed by Thaís Vieira (TSE)_

**Pull request merged**